appeals/api/src/server/endpoints/integrations/integrations.validators.js

```
@@ -0,0 +1,41 @@
+import { readdirSync, readFileSync } from 'node:fs';
+import Ajv from 'ajv';
+import addFormats from 'ajv-formats';
+import { setCache, getCache } from '#utils/cache-data.js';
+
+const SCHEMA_PATH = './src/message-schemas';
+const loadSchemas = async () => {
+	const cacheKey = 'integration-schemas';
+	let schemas = getCache(cacheKey);
+	if (!schemas) {
+		schemas = await readdirSync(SCHEMA_PATH)
+			.filter((file) => file.endsWith('.schema.json'))
+			.map((file) => {
+				return JSON.parse(readFileSync(`./src/message-schemas/${file}`).toString());
+			});
+
+		setCache(cacheKey, schemas);
+	}
+	return schemas;
+};
+export const schemas = {
+	appellantCase: 'appellant-case-submission',
+	lpaQuestionnaire: 'questionnaire-submission',
+	document: 'document'
+};
+
+export const validateFromSchema = async (
+	/** @type {string} */ schema,
+	/** @type {any} */ payload
+) => {
+	const schemas = await loadSchemas();
+	const ajv = new Ajv({ schemas });
+	addFormats(ajv);
+
+	const validator = ajv.getSchema(`${schema}.schema.json`);
+	if (validator && !validator(payload)) {
```

Don't we need to return an error if there is no validator?